src/test/java/com/bandwidth/sqs/actions/ChangeMessageVisibilityBatchActionTest.java

```diff
@@ -0,0 +1,43 @@
+package com.bandwidth.sqs.actions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+
+import com.amazonaws.services.sqs.model.ChangeMessageVisibilityBatchRequest;
+import com.bandwidth.sqs.queue.buffer.entry.ChangeMessageVisibilityEntry;
+
+import org.junit.Test;
+
+import java.time.Duration;
+import java.util.Map;
+
+public class ChangeMessageVisibilityBatchActionTest {
+
+    private static final String QUEUE_URL = "http://domain.com/path";
+
+    private static final ChangeMessageVisibilityEntry ENTRY = ChangeMessageVisibilityEntry.builder()
+            .receiptHandle("receipt handle")
+            .newVisibilityTimeout(Duration.ZERO)
+            .build();
+
+    private static final Map<String, ChangeMessageVisibilityEntry> ENTRY_MAP =
+            new ImmutableMap.Builder<String,
+                    ChangeMessageVisibilityEntry>()
+                    .put("0", ENTRY)
+                    .put("1", ENTRY)
+                    .build();
```

Comment:
I believe this could be simplified to `ENTRY_MAP = ImmutableMap.of("0", ENTRY, "1", ENTRY)